### DISCUSSION TITLE
issue with version 0.4.14

### DISCUSSION THREAD

**Discussion by ViolaYing** [jokergoo/circlize#302] [Q&A]
Hi ,
I am having an issue with circlize version 0.4.14, when using circos.genomicLabels() function, the labels are wired loacted along the chromosomes (hg38). I would like to know whether this issue can be fixed or where I could find and  re-install the package version 0.4.13?

See below the issue when plotting the labels. labels for chromosome 18 are very far away from the chromosome location, which was not like with the older version .

code for the labels:

```r
circos.par("track.height" = 0.05, cell.padding = c(0, 0, 0, 0))
circos.initializeWithIdeogram(cytoband="cytoBand_hg38.txt",species = "hg38")
set_track_gap(mm_h(0.3))
circos.genomicLabels(temp,
                     cex =0.2,
                     side = "inside",
                     labels.column = 4,
                     niceFacing = TRUE,
                     line_lwd=0.2,
                     connection_height = mm_h(5)
)
```

<img width="898" alt="test" src="https://user-images.githubusercontent.com/88832689/156741954-fdfe3d09-2345-4f61-9cf3-987fdef1e43d.png">

Thanks and wait for your reply.

Best,
Viola

**Comment by jokergoo** [comment]
See discussions at https://github.com/jokergoo/circlize/issues/300
